src/platform/plugins/shared/dev_tools/public/index.ts

```diff
@@ -8,7 +8,6 @@
  */
 
 // TODO: https://github.com/elastic/kibana/issues/110892
-/* eslint-disable @kbn/eslint/no_export_all */
```

Comment:
I wonder why we can remove this... isn't eslint looking at this dir anymore? @gsoldevila